src/index.js

```diff
@@ -49,33 +85,17 @@ const addTodo = (todoValue) => {
   const EditIcons = document.querySelectorAll('.fa-ellipsis-v');
   EditIcons.forEach((i) => {
     i.addEventListener('click', () => {
-      // addTodo(todoContainer, i.previousElementSibling);
+      editTodo(todoContainer, i.previousElementSibling);
     });
   });
-};
 
-// const editTodo = (todoContainer, todo) => {
-//   const editInput = document.createElement('input');
-//   editInput.type = 'text';
-//   editInput.classList = 'editInput';
-//   editInput.value = todo.textContent;
-//   todoContainer.replaceChild(editTodo, todo);
-//   editInput.addEventListener('keypress', (e) => {
-//     if (e.key === 'Enter') {
-//       const todoContainers = document.querySelectorAll('.todoContainer');
-//       const DataFromLocalStorage = JSON.parse(localStorage.getItem('list'));
-//       for (let i = 0; i < todoContainers.length; i += 1) {
-//         if (todoContainer[i].classList.contains('checkedContainer')) {
-//           DataFromLocalStorage[i].description = editInput.value;
-//           localStorage.setItem('list', JSON.stringify(DataFromLocalStorage));
-//         }
-//       }
-//       editInput.parentElement.classList.remove('checkedContainer');
-//       todoContainer.replaceChild(todo, editTodo);
-//       todo.textContent = editInput.value;
-//     }
-//   });
-// };
+  const removeIcons = document.querySelectorAll('.fa-trash-alt');
+  removeIcons.forEach((i) => {
+    i.addEventListener('click', () => {
+      removeTodo(i.parentElement);
+    });
+  });
+};
```

Comment:
Kindly, make sure that the user can `delete` all todos currently the last element is not begin deleted. Here I attached a video demo to show what is happening:



https://user-images.githubusercontent.com/34578631/172947093-b4c2fc6e-0ac6-466f-bf73-010f14687dd9.mov

Reply:
Hello @Mre55 
this option is sometimes working and sometimes doesn't work, can plz help with that?